build_tools/packaging/linux/build_package.py

```diff
@@ -146,7 +146,7 @@ def generate_changelog_file(pkg_info, deb_dir, config: PackageConfig):
         + config.version_suffix
     )
 
-    env = Environment(loader=FileSystemLoader("."))
+    env = Environment(loader=FileSystemLoader(str(SCRIPT_DIR)))
```

Comment:
Can you define `SCRIPT_DIR` in this file, instead of pulling it from `from packaging_utils import *`?

Reply:
Done